hail/python/test/hailtop/batch/test_regenie.py

```diff
@@ -72,11 +77,166 @@ def test_regenie_1pheno(self):
     @unittest.skipIf(not which("docker"), "docker command is missing")
     def test_regenie_nosplit(self):
         args = br.parse_input_args(["--local", "--step1", "example/step1.txt", "--step2",
-                                    "example/step2-nosplit.txt", "--outdir", self.outdir])
-        br.run(args)
+                                    "example/step2-nosplit.txt"])
+        br.run(**args)
 
-        out1 = f"{self.outdir}/{self.step2_out_prefix}.regenie"
+        out1 = f"{self.step2_out_prefix}.regenie"
 
         assert len(read(out1)) > 0
 
         rmtree(self.outdir)
+
+
+class ServiceBackendTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        os.system("git clone --depth 1 --branch v1.0.5.6 https://github.com/rgcgithub/regenie.git")
+        cls.step2_out_prefix = 'test_bin_out_firth'
+
+        cls.bucket_name = get_user_config().get('batch', 'bucket')
+
+        input_folder = 'batch-tests/resources/regenie-v1.0.5.6'
+        cls.gcs_input_dir = f'gs://{cls.bucket_name}/{input_folder}'
+
+        in_cluster_key_file = '/test-gsa-key/key.json'
```

Comment:
This is controlled by the `HAIL_GSA_KEY_FILE` environment variable, we should not hard code it here.